demo/js/demo.js

```diff
@@ -73,50 +64,140 @@ const Example = () => (
                 </AccordionItemTitle>
                 <AccordionItemBody>
-                    <p>
-                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc
-                        sollicitudin, quam at pulvinar rhoncus, turpis sem condimentum
-                        purus, id ullamcorper sem lectus nec nulla. Vestibulum fringilla
-                        nisi eget erat iaculis mollis a sit amet metus. Donec nec tortor
-                        mollis, scelerisque ex quis, pharetra elit. Nam leo nulla, porttitor
-                        at felis non, mollis blandit nisl. Curabitur finibus ex ut ipsum
-                        scelerisque, eget imperdiet lacus dignissim. Donec a blandit
-                        sem. Aliquam interdum posuere mi eu posuere. Nam venenatis
-                        auctor fermentum. Donec facilisis eleifend viverra. Integer malesuada
-                        dapibus sodales. Integer mollis maximus bibendum. Maecenas varius
-                        nisl aliquam velit commodo, quis consectetur massa fringilla. Pellentesque
-                        blandit ex mauris, vel vulputate mauris egestas vitae.
-                    </p>
+                    <table>
+                        <thead>
+                            <tr>
+                                <th>name</th>
+                                <th>type</th>
+                                <th>default</th>
+                                <th>description</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <td>accordion</td>
+                                <td>Boolean</td>
+                                <td>true</td>
+                                <td>Open only one item at a time or not</td>
+                            </tr>
+                            <tr>
+                                <td>onChange</td>
+                                <td>Function(keys)</td>
+                                <td>noop</td>
+                                <td>Triggered on change (open/close items)</td>
+                            </tr>
+                            <tr>
+                                <td>className</td>
+                                <td>String</td>
+                                <td>accordion</td>
+                                <td>CSS class(es) applied to the component</td>
+                            </tr>
+                        </tbody>
+                    </table>
                 </AccordionItemBody>
             </AccordionItem>
             <AccordionItem>
                 <AccordionItemTitle>
-                    <h3 className=" u-position-relative u-margin-bottom-s">
-                        Complex title
+                    <h3 className="u-position-relative">
+                        AccordionItem
                         <div className="accordion__arrow" role="presentation" />
```

Comment:
This arrow seems like a common need for accordions – should it also be a component available as part of this project?